gsa/src/web/pages/hosts/detailspage.js

```diff
@@ -164,13 +164,24 @@ const Hop = styled.div`
 `;
 
 const Details = ({entity, ...props}) => {
-  const {details = {}, routes = [], severity} = entity;
+  const {details = {}, identifiers = [], routes = [], severity} = entity;
+
   const os_cpe = isDefined(details.best_os_cpe)
     ? details.best_os_cpe.value
     : undefined;
   const os_txt = isDefined(details.best_os_txt)
     ? details.best_os_txt.value
     : undefined;
+
+  const bestOsMatchingIdentifier = identifiers.find(
```

Comment:
Maybe that's something for the being in the model already?

Reply:
Btw. please check if a core-js import is necessary for IE 11 find support.

Reply:
I thought about putting it in the model, but as long as we need this info only in one place, I think it makes more sense to load it here rather than for all hosts everywhere in GSA.